bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SortedLedgerStorage.java

```diff
@@ -310,16 +355,19 @@ public void onSizeLimitReached(final Checkpoint cp) throws IOException {
         scheduler.execute(new Runnable() {
             @Override
             public void run() {
-                try {
-                    LOG.info("Started flushing mem table.");
-                    interleavedLedgerStorage.getEntryLogger().prepareEntryMemTableFlush();
-                    memTable.flush(SortedLedgerStorage.this);
-                    if (interleavedLedgerStorage.getEntryLogger().commitEntryMemTableFlush()) {
-                        interleavedLedgerStorage.checkpointer.startCheckpoint(cp);
+                for (InterleavedLedgerStorage s : interleavedLedgerStorageList) {
+                    try {
+                        LOG.info("Started flushing mem table.");
```

Comment:
This line will be logged multiple times. Can we log something about 's'?

Reply:
finished:
print log's dir